packages/documentation/src/stories/getting-started/mission-statement/mission-statement.docs.mdx

```diff
@@ -0,0 +1,67 @@
+import { Meta } from '@storybook/blocks';
+import * as MissionStatementStories from './mission-statement.stories';
+import { roles, stakeholders, teamMembers } from './mission-statement.blocks';
+import './mission-statement.styles.scss';
+
+<Meta of={MissionStatementStories} />
+
+# Swiss Post Mission
+
+The Swiss Post Design System enables project teams to create seamless user experiences across all digital touchpoints by providing patterns, guidance and tools.
+
+## Who Uses the Design System?
+
+<ul className="list--roles d-flex m-0 p-0">
+  {roles.map((role, index) => (
+    <li key={index} className="list--roles__item mb-3 p-3 h-100">
+      <strong>{role.role}</strong>
+      <div className="mt-2">{role.description}</div>
+    </li>
+  ))}
+</ul>
+
+### The Design System is well represented and supported by our stakeholders, including:
+
+<ul className="list-group">
+  {stakeholders.map((stakeholder, index) => (
+    <li key={index} className="list-group-item w-sm-100 w-md-50 w-lg-25">{stakeholder.role}</li>
+  ))}
+</ul>
+
+### Documentation
+
+Entry points to the Design System are public. All documentation is written in English and uses common terminology. The documentation platforms are in sync with implementations and cross-linked. Regular updates ensure that the documentation stays aligned with the latest changes and best practices.
+
+### Core Team
+
+<ul className="list-group list-group-horizontal-md mb-3 p-0 justify-content-center-lg align-items-center-lg">
+  {teamMembers.map((teamMember, index) => (
+    <li key={index} className="list-group-item py-4 gap-3 fw-bold display-flex flex-lg-column w-md-25 justify-content-center">
+      <post-icon name={teamMember.icon} style={{ fontSize: '3rem' }}></post-icon>
+      {teamMember.name}
+    </li>
+  ))}
+</ul>
+
+These contributors are responsible for major implementations and play a central role in shaping the system. To ensure meaningful input, they have allocated time and budget to actively participate in the project, both in Figma and Web development.
+
+### Quality & Compliance
+
+People are using the Design System because it has a low barrier of entry, excellent documentation, is easy to use and improves the time to market for their product significantly. It fulfills the standards of the Web Content Accessibility Guidelines (WCAG), Level AA.
+
+<div className="d-flex align-items-center mt-4 gap-2">
+  <post-icon name="2022" style={{ fontSize: '2rem' }}></post-icon>
+  <p>Onboarding is mandatory for external agencies.</p>
+</div>
+
+<p className="mt-4 mb-1 text-align-start">
+  The quality of the Design System is being measured by pre-defined KPIs and is constantly improved.
+</p>
+
+<div className="progress-system-quality">
```

Comment:
The progress bar is a nice idea, but since we don't have the data to make it accurate at the moment, it might be better to remove it for now.